web/src/pages/support/login.tsx

```diff
@@ -0,0 +1,108 @@
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { SingleColumnContainer } from "@/components/njwds/SingleColumnContainer";
+import { AuthContext } from "@/contexts/authContext";
+import { IsAuthenticated } from "@/lib/auth/AuthContext";
+import { configureAmplify, triggerSignOut } from "@/lib/auth/sessionHelper";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import analytics from "@/lib/utils/analytics";
+import { NextSeo } from "next-seo";
+import { ReactElement, useContext } from "react";
+
+const LoginSupportPage = (): ReactElement => {
+  const { Config } = useConfig();
+  const loginSupport = Config.loginSupportPage;
+  const { state: authState } = useContext(AuthContext);
+
+  const isLoggedIn = authState.isAuthenticated === IsAuthenticated.TRUE;
+
+  const customComponents = {
+    logoutButton: (
+      <UnStyledButton
+        onClick={async () => {
+          configureAmplify();
+          await triggerSignOut();
+        }}
+        className="logout-button-unstyled"
+      >
+        {loginSupport.logoutButtonText}
+      </UnStyledButton>
+    ),
+  };
+
+  return (
+    <>
+      <NextSeo
+        title={loginSupport.pageTitle}
+        description="Step-by-step guide to help you log in to Business.NJ.gov using your myNewJersey account."
+        noindex={true}
+      />
+      <PageSkeleton showNavBar isSeoStarterKit={!isLoggedIn} isLoginPage={isLoggedIn}>
+        <main className="usa-section padding-top-1" id="main">
+          <div className="grid-container-widescreen desktop:padding-x-7 bg-cool-extra-light padding-y-2">
+            <Heading level={1} className="text-secondary-darker text-left">
+              {loginSupport.pageTitle}
+            </Heading>
+            <p className="text-base-dark ">Last updated {loginSupport.lastUpdated}</p>
+          </div>
+          <SingleColumnContainer>
+            <div className="padding-top-1 desktop:padding-top-0 login-support-content">
+              <Content>{loginSupport.multipleAccountsCallout}</Content>
+              <Content showLaunchIcon={false}>{loginSupport.myNewJerseyIntroContent}</Content>
+              <ol className="padding-top-1">
+                {loginSupport.steps.map((step, index) => (
+                  <li key={index} className="margin-y-05">
+                    <Content showLaunchIcon={false}>{step.content}</Content>
+                    {step.imagePath && (
+                      <div className="margin-top-2 margin-bottom-4">
+                        <img
+                          src={step.imagePath}
+                          alt={step.imageAlt || ""}
+                          className="width-full maxw-tablet border-1px border-base-lighter"
+                        />
+                      </div>
+                    )}
+                  </li>
+                ))}
+              </ol>
```

Comment:
@jphechter updated this to make the ordered list more explicit. Now each element of the [steps array](https://github.com/newjersey/navigator.business.nj.gov/pull/11876/changes#diff-6cd9d1c05202e85ce783d7ddeec13bbe90102c6cdb2e8ddffb55cd732439d181R7-R37) corresponds to a single `<li>`, with the image property optional